src/ray/raylet/node_manager.cc

```diff
@@ -622,6 +622,10 @@ void NodeManager::AssignTask(Task &task) {
       // There are no more non-actor workers available to execute this task.
       // Start a new worker.
       worker_pool_.StartWorker();
+    } else if (spec.IsActorCreationTask()) {
```

Comment:
Don't actor creation tasks already go through the `if` statement, so they won't hit the `else if`?